tests/run-all-tests.sh

```diff
@@ -1,9 +1,12 @@
 #!/bin/bash
 
+# example override: "coverage run -a", or (later) "python3"
+INTERP=${1:-python}
+
 RES=0
 for f in $(find . \( -path "./xcp" -prune \) -o -name "test*.py" -print)
 do
     echo "Running $f"
-    PYTHONPATH=.. python $f || RES=1
+    PYTHONPATH=.. $INTERP $f || RES=1
```

Comment:
While we are here:
* using `set -e`, `set -o pipefail` is good practice
* `PYTHONPATH=.. $INTERP "$f"` would help in the (unlikely) case of a space in `$f`. This could also apply to `$INTERP` if we assume that no flags are passed by just a command or path.

Reply:
New version of the script should be better, only a wrapper to pytest that sets PYTHONPATH.  I'd be happy to even get rid of it, but this PYTHONPATH problem should be handled first.
One way would be to work on the project (including running tests) using `pip install -e .` to get its module installed in the environment, but it does not install anything (finds the dev code as a valid "installed version", which pytest does not).